server/tck/src/test/resources/tests/context/output.json

```diff
@@ -12,4 +12,4 @@
       "path": "/contextMethodInjectedFromSource/path"
     }
```

Comment:
Here we also need to check the value of `asyncOperationType`. (Should be QUERY if the context propagated)

Reply:
should be resolved by https://github.com/smallrye/smallrye-graphql/pull/558/commits/792928a7bd0b6656db77d0087d3142ed46d1f89a